pkg/cvo/cvo_scenarios_test.go

```diff
@@ -1218,18 +1196,25 @@ func TestCVO_UpgradeUnverifiedPayload(t *testing.T) {
 			t.Fatalf("saw too many sync events of the wrong form")
 		}
 	}
+	// wait until the new payload is applied
+	count = 0
+	for {
+		var status SyncWorkerStatus
+		select {
+		case status = <-worker.StatusCh():
+		case <-time.After(3 * time.Second):
+			t.Fatalf("never saw expected apply event")
+		}
+		if status.Step == "ApplyResources" {
```

Comment:
you're dropping the `ApplyResources` step.  Makes sense to me, but we'll need some logic change here.

Reply:
`ApplyResources` is completely gone in bc3998ad51474fb; marking resolved.